src/window.rs

```diff
@@ -682,6 +682,19 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused. Providing `None` will unset the requesting of user attention.
```

Comment:
It should be clear to the user if they are expected to call the unset option, but this seems to be already in discussion.

Reply:
Added extra documentation regarding this!